docs/source/guide/get_started/installation.rst

```diff
@@ -65,6 +65,68 @@ virtual environment.
 5. Once the package is installed in the virtual environment, you can use full
 OpenVINO™ Training Extensions command line functionality.
 
+*************************************************************
+Install OpenVINO™ Training Extensions for users (XPU devices)
+*************************************************************
+
+1. Follow the first two steps from above instructions
+on cloning the repository and creating a virtual environment.
+
+2. Install Intel Extensions For Pytorch (IPEX).
+Follow the `official documentation <https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.10%2Bxpu>`_ to install prerequisites such as OneAPI and proper drivers.
+
+.. code-block:: shell
+
+    python -m pip install torch==2.1.0a0 torchvision==0.16.0a0 torchaudio==2.1.0a0 intel-extension-for-pytorch==2.1.10+xpu --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+
+3. Install MMCV.
+It is required to install mmcv from source to properly build it with IPEX.
+
+.. code-block:: shell
+
+    git clone https://github.com/open-mmlab/mmcv
+    cd mmcv
+    git checkout v2.1.0
+    MMCV_WITH_OPS=1 pip install -e .
+
+4. Install OpenVINO™ Training Extensions
+package from either:
+
+* A local source in development mode
+
+.. code-block:: shell
+
+    pip install -e .
+
+* PyPI
+
+.. code-block:: shell
+
+    pip install otx
+
+5. Install requirements for training
+excluding Pytorch and MMCV.
+
+.. code-block:: shell
+
+    otx install -v --do-not-install-torch --do-no-install-mmcv
```

Comment:
It seems that if passing `--do-no-install-mmcv`, otx doesn't install all mmXX frameworks including mmpretrain, mmdet and etc. Could you check it? In my case, installation w/o `otx install -v --do-not-install-torch` even works well.

Reply:
Yesterday when I tried to install following my instructions - everything was installed properly. Installing mmcv from mim leads to the same error with mmcv.ops
I will check about other mmXX packages

Reply:
```
    mmcv_install_args = []
    if mmcv_requirements and not do_not_install_mmcv:
        mmcv_install_args = get_mmcv_install_args(torch_requirement, mmcv_requirements)
        install_args += ["openmim"]

    # Install requirements.
    with console.status("[bold green]Working on installation...\n") as status:
        if verbose:
            logger.setLevel(logging.INFO)
            status.stop()
        console.log(f"Installation list: [yellow]{install_args}[/yellow]")
        status_code = create_command("install").main(install_args)
        if status_code == 0:
            console.log(f"Installation Complete: {install_args}")

        # https://github.com/Madoshakalaka/pipenv-setup/issues/101
        os.environ["SETUPTOOLS_USE_DISTUTILS"] = "stdlib"

        # Install mmX requirements if the task requires mmX packages using mim.
        if mmcv_install_args and status_code == 0:
            console.log(f"Installation list: [yellow]{mmcv_install_args}[/yellow]")
            status_code = mim_installation(mmcv_install_args)
            if status_code == 0:
                console.log(f"MMLab Installation Complete: {mmcv_install_args}")
```
This is "src/otx/cli/install.py". It seems that if `do_not_install_mmcv` is True, `mmcv_install_args` will be `[]`and then `if mmcv_install_args and status_code == 0` can't be exceuted.

Reply:
Actually, you are right. Also, I noteiced, that if correct version of mmcv is installed, it is not collected during parsing "mmcv_install_args ". So, let's just remove "do_not_install_mmcv". I can confirm that OTX2.0 can be installed w/o problems omitting this step